libpod/container_api.go

```diff
@@ -555,7 +555,7 @@ func (c *Container) WaitForExit(ctx context.Context, pollInterval time.Duration)
 				// The container never ran.
 				return true, 0, nil
 			}
-			return true, -1, err
+			return true, -1, fmt.Errorf("%v (container in state %s)", err, c.state.State)
```

Comment:
Wrap the error with `%w`, just in case there are callers that want to check the cause.

Reply:
Great catch, thank you!